src/space-between/internal.tsx

```diff
@@ -24,8 +28,11 @@ export default function InternalSpaceBetween({
    */
   const flattenedChildren = flattenChildren(children);
 
+  const ParentTag = asList ? 'ul' : 'div';
```

Comment:
allowing `aria-setsize` to be set here (e.g. when tokens are hidden) might be needed, based on discussions

Reply:
Right, my plan is to change `asList` to `listProps` going forward as soon any list-specific property is required.